monai/networks/nets/daf3d.py

```diff
@@ -0,0 +1,574 @@
+            can be used to visualize inner functionality of the network
+    """
+
+    def __init__(self, in_channels, out_channels, visual_output=False):
+        super().__init__()
+        self.visual_output = visual_output
+        self.backbone_with_fpn = Daf3dBackboneWithFPN(
+            backbone=Daf3dBackbone(in_channels),
+            return_layers={"layer1": "feat1", "layer2": "feat2", "layer3": "feat3", "layer4": "feat4"},
+            in_channels_list=[256, 512, 1024, 2048],
+            out_channels=128,
+            spatial_dims=3,
+        )
+        self.predict1 = nn.Conv3d(128, out_channels, kernel_size=1)
+
+        group_norm = ("group", {"num_groups": 32, "num_channels": 64})
+        act_prelu = ("prelu", {"num_parameters": 1, "init": 0.25})
+        self.fuse = nn.Sequential(
+            Convolution(
+                spatial_dims=3,
+                in_channels=512,
+                out_channels=64,
+                kernel_size=1,
+                adn_ordering="NA",
+                norm=group_norm,
+                act=act_prelu,
+            ),
+            Convolution(
+                spatial_dims=3,
+                in_channels=64,
+                out_channels=64,
+                kernel_size=3,
+                adn_ordering="NA",
+                padding=1,
+                norm=group_norm,
+                act=act_prelu,
+            ),
+            Convolution(
+                spatial_dims=3,
+                in_channels=64,
+                out_channels=64,
+                kernel_size=3,
+                adn_ordering="NA",
+                padding=1,
+                norm=group_norm,
+                act=act_prelu,
+            ),
+        )
+        self.attention = AttentionModule(
+            spatial_dims=3, in_channels=192, out_channels=64, norm=group_norm, act=act_prelu
+        )
+
+        self.refine = Convolution(3, 256, 64, kernel_size=1, adn_ordering="NA", norm=group_norm, act=act_prelu)
+        self.predict2 = nn.Conv3d(64, out_channels, kernel_size=1)
+        self.aspp = Daf3dASPP(
+            spatial_dims=3,
+            in_channels=64,
+            conv_out_channels=64,
+            out_channels=64,
+            kernel_sizes=(3, 3, 3, 3),
+            dilations=((1, 1, 1), (1, 6, 6), (1, 12, 12), (1, 18, 18)),  # type: ignore
+            norm_type=group_norm,
+            acti_type=None,
+            bias=True,
+        )
+
+    def forward(self, x):
+        # layers from 1 - 4
+        single_layer_features = list(self.backbone_with_fpn(x).values())
+
+        # first 4 supervised signals (SLFs 1 - 4)
+        supervised1 = [self.predict1(slf) for slf in single_layer_features]
+
+        mlf = self.fuse(torch.cat(single_layer_features, 1))
+
+        attentive_features_maps = [self.attention(slf, mlf) for slf in single_layer_features]
+        att_features, att_maps = zip(*attentive_features_maps)
```

Comment:
```suggestion
        att_features, att_maps = tuple(zip(*attentive_features_maps))
```
this change seems to fix the torchscript test and I don't know why, maybe it's a bug from torchscript itself.. @Al3xand1a

Reply:
so this still doesn't work, it's not easy to make it fully compatible with torchscript in this case, so I'm adding a line to skip the torchscript test.